ci/common_install.sh

```diff
@@ -122,7 +122,19 @@ fi
 
 # Setup ccache
 if [ "$B2_USE_CCACHE" == "1" ]; then
-    "$CI_DIR"/setup_ccache.sh
+    if ! "$CI_DIR"/setup_ccache.sh 2>&1; then
+        set +x
+        echo
+        printf '=%.0s' {1..120}
+        echo
+        echo "Failed to install & setup ccache!"
+        echo "Will NOT use CCache for building."
+        printf '=%.0s' {1..120}
+        echo
+        echo
+        B2_USE_CCACHE=0
```

Comment:
I'm guessing this does not need to be pushed into the GITHUB_ENV because it's only used within the same step?

Reply:
Correct. I was thinking about it but it really is only used here to "replace" the real compiler by the wrapper command in the Jam config file.